pydatalab/pydatalab/routes/v0_1/users.py

```diff
@@ -22,13 +24,13 @@ def save_user(user_id):
 
     update = {}
 
+    if display_name:
+        update["display_name"] = DisplayName(display_name)
+
     if contact_email or contact_email in (None, ""):
         if contact_email == "":
             contact_email = None
-        update["contact_email"] = contact_email
-
-    if display_name:
-        update["display_name"] = display_name
+        update["contact_email"] = EmailStr(contact_email)
```

Comment:
I think we should probably catch and return `ValueError` on both of these calls, then simply alert whether it is the display name or contact email that is "bad" -- this alert can then be shown in the UI.

I think this is the one change that we should get in before merging, as at the moment I think you just get an incomprehensible message back